### ISSUE TITLE
Show multiline's answer in new line

### ISSUE THREAD

**Issue by AielloChan** [AlecAivazis/survey#336]
Now, it looks wired:

![image](https://user-images.githubusercontent.com/7900765/109301804-653cfd00-7873-11eb-81cc-2b36aa7d5e54.png)

How about showing the answer in a new line, just like below:

![image](https://user-images.githubusercontent.com/7900765/109302731-a681dc80-7874-11eb-85f2-457c1e146fff.png)

**Comment by AielloChan** [comment]
And it just needs and a `\n` in the template.

![image](https://user-images.githubusercontent.com/7900765/109304197-dd58f200-7876-11eb-95ee-9257b39b7fe8.png)


https://github.com/AielloChan/survey/commit/44148ffac0f0a13e6031fb046338aff69e01a889

**Comment by AielloChan** [comment]
So, any idea?

**Comment by AlecAivazis** [comment]
Thanks for the ping @AielloChan! It's been hard to wrangle all of the different notifications I get so I appreciate your patience.

At the moment, a change like this would be considered a break in the API. `survey` is used by many large companies who run integration tests and a change like this would inevitably make those fail.

I know its not the answer you or the others want to hear but I dont think we can change this at the moment. I will leave the issue open and tag it for future consideration though!

**Comment by AielloChan** [comment]
> Thanks for the ping @AielloChan! It's been hard to wrangle all of the different notifications I get so I appreciate your patience.
> 
> At the moment, a change like this would be considered a break in the API. `survey` is used by many large companies who run integration tests and a change like this would inevitably make those fail.
> 
> I know its not the answer you or the others want to hear but I dont think we can change this at the moment. I will leave the issue open and tag it for future consideration though!

Okay, this would be the best choice for now. 😄

**Comment by graugans** [comment]
I agree with @AielloChan the current behavior bugged me  as-well. I am just staring with Go so am I far from being one which could provide any implementation advice nor I fully understand how survey is internally working.

Given the templating engine of Go and an extension of [PromptConfig](https://github.com/AlecAivazis/survey/blob/099a96894e7a1941cca62b6a9f1a06dadf8664ae/survey.go#L113) it should be possible to flag this feature. So no integration test on larger companies will break

**Comment by silvanbrummen** [comment]
I'm running into this styling issue myself as well currently. The solution proposed by @graugans seems like a good option to me. What do you think @AlecAivazis?

**Comment by CastawayEGR** [comment]
+1

**Comment by sheldonhull** [comment]
@AlecAivazis would a PR just adding that missing line break to the go template be acceptable? I don't mind submitting that. I noticed pterm input and some other tools do this so it's only the survey library that I've noticed the multiline editing on the same line happen in. It's a minor "nit" but it's sufficiently jarring to users that I figured I'd see if I could help get this through.
